### PR TITLE
Granted Admin users ability to impersonate member

### LINKED ISSUE #12126: Grant Administrator users ability to impersonate members
### Issue Summary

Logging in as the Owner user is currently the only way to impersonate a member, but this should also be possible as an Administrator user.

Viewing a member page as the Owner user, the _Impersonate_ button is present:
Screenshot 2020-08-13 at 15 57 04

Viewing the same page logged in as an Administrator user, the _Impersonate_ button is **not** present:
Screenshot 2020-08-13 at 15 58 21

### PR DESCRIPTION
**Pull request by rshbhgrg**
refs https://github.com/TryGhost/Ghost/issues/12126

- Adds migration to add impersonation permission to administrators
- Adds default permission fixture for administrators to read member impersonation urls
- Allows administrators to create member impersonation magic links

### CODE REVIEW

**Comment by rishabhgrg** [comment]
@allouis @naz @daniellockyer Need 👀 on the small new [migration](https://github.com/TryGhost/Ghost/pull/12132/files#diff-a681e6a5405bb05bc135ba4c7afd691a) which adds member impersonation permission to Administrator users which needs :)

File: core/server/data/migrations/versions/3.30/01-add-member-signin-url-permission-roles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const {
+    addPermissionToRole,
+    combineTransactionalMigrations
+} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
```
Comment (allouis) [inline core/server/data/migrations/versions/3.30/01-add-member-signin-url-permission-roles.js:6]: No need to combine any migrations here as you only have!

**Review by allouis — COMMENTED** [review]
Just a small comment about the use of `combineTransactionalMigrations` :relaxed:

**Review by naz — APPROVED** [review]
Love how little code we now need to handle new permissions :eyes: :dash: :+1:

File: core/server/data/migrations/versions/3.30/01-add-member-signin-url-permission-roles.js (hunk 1 above)
Comment (rishabhgrg) [reply core/server/data/migrations/versions/3.30/01-add-member-signin-url-permission-roles.js]: @allouis Aah yeah 🤦 I was not sure if by default individual call created a transactional migration but just double checked and it makes sense. Updated now :)

**Review by allouis — APPROVED** [review]

**Review by daniellockyer — APPROVED** [review]

**Pull request merged**